azext_edge/edge/providers/check/akri.py

```diff
@@ -67,15 +67,11 @@ def evaluate_core_service_runtime(
     check_manager = CheckManager(check_name="evalCoreServiceRuntime", check_desc="Evaluate Akri core service")
 
     padding = 6
-    akri_runtime_resources: List[dict] = []
-    for prefix in AKRI_PREFIXES:
-        akri_runtime_resources.extend(
-            get_namespaced_pods_by_prefix(
-                prefix=prefix,
-                namespace="",
-                label_selector="",
-            )
-        )
+    akri_runtime_resources = get_namespaced_pods_by_prefix(
```

Comment:
Is there a label selector that can be applied ? Calling `get_namespaced_pods_by_prefix` without that is getting all pods and filtering client side

Reply:
new pr coming in soon 

(I had to recheck the function to make sure it wouldn't theoretically fetch the same pod)